### ISSUE TITLE
Question about difficulty on multiple different hashrates

### ISSUE THREAD

**Issue by freddya747** [fireice-uk/xmr-stak#1104]
Hi guys,

I got a system running an AMD TR 1950x @ 4.05ghz + 4 AMD Vega FE 16GB on Windows Server 2016 with unified xmr-stak.
The CPU is using 15 threads for mining and 1 thread for GPU affine_to_cpu.

Hashrates in xmr-stak: ~1400H CPU (15 x 94H) & ~8320H GPUs (4 x 2080H) ==~9700H/s

I mined multiple weeks in different public pools, all using nodejs-pool as server, but could never reach a "24 Hour Avg. Hashrate" above 9000H/s on any pool I tried for at least 3 days.
The system is monitored 24/7 without having any issues, my 15min avarage never went below 9600H/s and never had thermal throttling.
I became suspicious and set up a nodejs-pool myself to check if there any changes on the hashrates, but got same results when using VARDIFF.

So, to my question: 
Is this a difficulty problem caused by "extremely" different hashrates per xmr-stack ID?
94H/CPU-thread vs 1040H/GPU-thread (using 2 GPU threads per card)
In other words, would it be better to use xmr-stak-cpu as 1st worker and xmr-stak-amd as 2nd worker to get different difficulties that are more appropriate to their hashes per thread?
Or is it normal that all pools only get like 9/10 of my hashrate?

Thanks in advance

**Comment by freddya747** [comment]
Tried it with xmr-stak running 2 times, one CPU only and one GPU only
Had about 200Hashes less, so running just 1 instance of xmr-stak is faster
